packages/rum-core/src/performance-monitoring/capture-navigation.js

```diff
@@ -356,11 +382,11 @@ function captureNavigation(transaction) {
      * Capture resource timing information as spans
      */
     const resourceEntries = PERF.getEntriesByType(RESOURCE)
-    const apiCalls = getApiSpanNames(transaction)
+    const apiSpans = getAPISpans(transaction)
```

Comment:
TODO: Skip this filtering logic for transactions other than `page-load` as we know our script would have kicked in and patched the necessary methods.